Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -17,14 +17,90 @@ allprojects {
       flatDir {
         dirs 'libs'
       }
+
+      // Uncomment the repositories for the third parties you wish to integrate
```

Comment:
don't forget to revert all the changes here as well

Reply:
I removed it in this commit
5e0edf1